javascript/javascript1/week3/itemRemoval.js

```diff
@@ -0,0 +1,17 @@
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+];
+const nameToRemove = "Ahmad";
+
+while (names.includes(nameToRemove)) {
```

Comment:
you don't need to use while here, because includes function iterate to array and check the value if it includes inside array it returns true otherwise false. I mean like
```
if (names.includes(nameToRemove)) {
    names.splice(names.indexOf(nameToRemove),1);
}
```
you can check this link for more [info:`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)

Reply:
Used while because if it contains same name more times, so that it can remove all the values with that name in the array.